preproc_script_1.sh

```diff
@@ -62,13 +69,14 @@ done
 
 # Evaluating set options
 OPTIND=1
-while getopts "hm:b:s:f:d:" opt
+while getopts "hm:b:s:c:f:d:" opt
 do
   case "$opt" in
     "h") usage; exit 0 ;;
     "m") model="${OPTARG}" ;;
     "b") bet_f="${OPTARG}" ;;
     "s") NeedSTC="${OPTARG}" ;;
+	"c") NeedDC="${OPTARG}" ;;
```

Comment:
This should be indented at the same level as the other options.

Reply:
Thanks! fixed.